src/cmp-ui.ts

```diff
@@ -29,4 +59,4 @@ export const setupMessageHandlers = (
     window.addEventListener('message', receiveMessage, false);
 };
 
-export const canShow = (): boolean => !readGuCookie() || !readIabCookie();
+export const canShow = (): boolean => !readIabCookie(); // TODO: Restore readGuCookie check once we start saving GU cookie
```

Comment:
What does this TODO mean?

Reply:
As mentioned previously i've left this in as a marker that once we start saving the "guCookie" with the PECR purposes we'll want to include `readGuCookie()` in the `canShow` check.